Resources/views/admin/fields/file-link-multiple.blade.php

```diff
@@ -1,54 +1,21 @@
 <script src="{{ Module::asset('dashboard:vendor/jquery-ui/jquery-ui.min.js') }}"></script>
```

Comment:
The one from config is loaded only when composer view pattern is met ``['media::admin.fields.new-*']``.
I was thinking that loading only that without whole media-partial.js would be correct one. Since user probably will not mix new and old styles of adding media.
So for old it loads that jquery old way, and for new there is assetPipeline